.github/PULL_REQUEST_TEMPLATE/my_jumia.md

```diff
@@ -0,0 +1,18 @@
+# Proposed Changes
+
+-
+-
+-
+
+## MyJumia PR checklist
+
+- [ ] Behavior changes are covered by new tests
+  - [ ] functional tests
+  - [ ] unit tests
+  - [ ] e2e tests
```

Comment:
I would just change the order to sound more natural, in terms of the "pyramid". Unit tests, API tests and functional tests.
Consider renaming from "e2e" to API tests. e2e and functional tests may sound like they are the same.

Reply:
Done